Frameworks/UIKit/UIView.mm

```diff
@@ -724,29 +727,34 @@ - (void)updateAccessibility {
 /**
  @Status Interoperable
 */
-+ (instancetype)allocWithZone:(NSZone*)zone {
-    UIView* ret = [super allocWithZone:zone];
-
+- (instancetype)initWithFrame:(CGRect)frame {
     // Run on the main thread because the underlying XAML objects can only be
     // called from the UI thread
     RunSynchronouslyOnMainThread(^{
```

Comment:
>RunSynchronouslyOnMainThread [](start = 4, length = 28)

Should we have called [super init] preceded this as well? UIView -> UIResponder -> NSObject? I noticed that UIWebView has this pattern.